### ISSUE TITLE
[Feature Request] Missing the inventory management hotkeys

### ISSUE THREAD

**Issue by DannyDan4** [jeremiahwinsley/InvTweaksRenewed#8]
It appears this is missing the old inventory management key combos from the old inventory tweaks, such as alt+click to throw a stack, control+shift+click to move all items of a type into a container, space+click to move every item from a container, and combos of which, control+alt+shift+click to throw every item of a type, or space+alt+click to throw every item from a container.

**Comment by desagas** [comment]
Yeah, that is the only reason I used this mod at all. It is the main feature I looked for, and am sadly disappointed until it comes back.
